### DISCUSSION TITLE
To fix: nightly version there's not correct related document id

### DISCUSSION THREAD

**Discussion by njunior1980** [ravendb/ravendb#16346] [General]
![image](https://user-images.githubusercontent.com/5822802/234130798-5ca80e0e-54c8-4629-9808-0517555538bc.png)

![image](https://user-images.githubusercontent.com/5822802/234130903-958439b3-5aed-4527-918a-424ce4fa6c07.png)

**Comment by ml054** [comment]
Hi, 

Thanks for letting us know. 

I'd opened ticket in our bug tracker: https://issues.hibernatingrhinos.com/issue/RavenDB-20382/To-fix-nightly-version-theres-not-correct-related-document-id
